prow/jobs/kyma-project/runtime-watcher/runtime-watcher.yaml

```diff
@@ -1,102 +1,4 @@
-        preset-sa-kyma-push-images: "true"
-        preset-image-builder-ado-token: "true"
-      always_run: true
-      skip_report: false
-      decorate: true
-      cluster: untrusted-workload
-      max_concurrency: 10
-      spec:
-        containers:
-          - image: "europe-docker.pkg.dev/kyma-project/prod/image-builder:v20240829-9a504e1b"
-            securityContext:
-              privileged: false
-              seccompProfile:
-                type: RuntimeDefault
-              allowPrivilegeEscalation: false
-            command:
-              - "/image-builder"
-            args:
-              - "--build-in-ado=true"
-              - "--name=runtime-watcher"
-              - "--config=/config/kaniko-build-config.yaml"
-              - "--context=runtime-watcher"
-              - "--dockerfile=Dockerfile"
-            resources:
-              requests:
-                memory: 8Gi
-                cpu: 2
-              limits:
-                memory: 16Gi
-                cpu: 4
-            volumeMounts:
-              - name: config
-                mountPath: /config
-                readOnly: true
-        volumes:
-          - name: config
-            configMap:
-              name: kaniko-build-config
 postsubmits:
-  kyma-project/runtime-watcher:
-    - name: main-runtime-watcher-build
-      annotations:
-        description: "build runtime watcher image with the latest tag"
-        owner: "jellyfish"
-      labels:
-        prow.k8s.io/pubsub.project: "sap-kyma-prow"
-        prow.k8s.io/pubsub.runID: "watcher-build-latest"
-        prow.k8s.io/pubsub.topic: "prowjobs"
-        preset-sa-kyma-push-images: "true"
-        preset-image-builder-ado-token: "true"
-      always_run: true
-      skip_report: false
-      decorate: true
-      cluster: trusted-workload
-      max_concurrency: 10
-      branches:
-        - ^main$
-      spec:
-        containers:
-          - image: "europe-docker.pkg.dev/kyma-project/prod/image-builder:v20240829-9a504e1b"
-            securityContext:
-              privileged: false
-              seccompProfile:
-                type: RuntimeDefault
-              allowPrivilegeEscalation: false
-            command:
-              - "/image-builder"
-            args:
-              - "--build-in-ado=true"
-              - "--name=runtime-watcher-skr"
-              - "--config=/config/kaniko-build-config.yaml"
-              - "--context=runtime-watcher"
-              - "--dockerfile=Dockerfile"
-              - "--tag=latest"
-            resources:
-              requests:
-                memory: 8Gi
-                cpu: 2
-              limits:
-                memory: 16Gi
-                cpu: 4
-            volumeMounts:
-              - name: config
-                mountPath: /config
-                readOnly: true
-        volumes:
-          - name: config
-            configMap:
-              name: kaniko-build-config
     - name: release-runtime-watcher-build
```

Comment:
do you still need this job? If yes then you can't remove line 50: https://github.com/kyma-project/test-infra/pull/11773/files#diff-d8b900be88d027f100639c5c702c735ce1960958b9bb8d3f674008b0e7c3292fL50

Reply:
Yes that was a mistake. Thanks! :D